src/main/java/com/minecolonies/coremod/commands/colonycommands/ListColoniesCommand.java

```diff
@@ -56,16 +69,37 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
         return super.getCommandUsage(sender);
     }
 
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final ActionMenu actionMenu) throws CommandException
+    {
+        final Integer page = actionMenu.getIntegerForArgument("page");
+        final Integer abandonedSinceTimeInHours = actionMenu.getIntegerForArgument("abandonedSinceTimeInHours");
+        executeShared(server, sender, page, abandonedSinceTimeInHours);
+    }
+
     @Override
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
     {
-        int page = getIthArgument(args, 0, 1);
-        final int abandonedSince = getIthArgument(args, 1, 0);
+        final int page = getIthArgument(args, 0, 1);
+        final int abandonedSinceTimeInHours = getIthArgument(args, 1, 0);
+
+        executeShared(server, sender, page, abandonedSinceTimeInHours);
+    }
+
+    private void executeShared(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, Integer page, Integer abandonedSinceTimeInHours) throws CommandException
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "server". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)